docs/docs/developers/guides/smart_contracts/writing_contracts/how_to_emit_event.md

```diff
@@ -46,13 +53,4 @@ Once emitted, unencrypted events are stored in AztecNode and can be queried by a
 
 ## Costs
 
-All event data is pushed to Ethereum as calldata by the sequencer and for this reason the cost of emitting an event is non-trivial.
-
-In the Sandbox, an encrypted note has a fixed overhead of 4 field elements (to broadcast an ephemeral public key, a contract address, and a storage slot); plus a variable number of field elements depending on the type of note being emitted.
-
-A `ValueNote`, for example, currently uses 3 fields elements (plus the fixed overhead of 4). That's roughly `7 * 32 = 224` bytes of information.
-
-#include_code value-note-def /noir-projects/aztec-nr/value-note/src/value_note.nr
-
-- There are plans to compress encrypted note data further.
-- There are plans to adopt EIP-4844 blobs to reduce the cost of data submission further.
+Event data is pushed to Ethereum, in blobs, by the sequencer and for this reason the cost of emitting an event can be non-trivial. As mentioned above, emitting encrypted events is optional and there will likely be alternative options for developers to choose from in the future.
```

Comment:
"for this reason the cost of emitting an event can be non-trivial" does it mean "_calculating_ the cost" can be non-trivial?

Reply:
I think calculating it should be trivial. It's just a matter of calculating the data usage cost for a given DA and calculating how much data an event uses. I expect there to be tools that do this

Reply:
I didnt understand what it meant by:  the cost is non-trivial. Unless it meant: the cost is non-negligible?